CMakeLists.txt

```diff
@@ -185,11 +185,60 @@ include_directories(${PROJECT_BINARY_DIR})
 #    PATH_VARS CMAKE_INSTALL_INCLUDEDIR
 #)
 
+
 #
 # codec2 library
 #
 add_subdirectory(src)
 
+
+if(UNIX)
+    target_link_libraries(codec2 m)
+endif(UNIX)
+
```

Comment:
The 3 lines above have to go to src/CMakeLists.txt behind add_library(codec2 ${CODEC2_SRCS}) then cmake works for me (and would in Travis)

Reply:
Ahh, you must be running an older version of CMake where it has to go in the same CMakeLists file... I'll fix it here in a second.